eng/pipelines/templates/stages/pr.platform-matrix.json

```diff
@@ -0,0 +1,10 @@
+{
+  "matrix": {
+      "$IMPORT": "eng/pipelines/templates/stages/platform-matrix.json"
+  },
+  "exclude": [
```

Comment:
Is the goal to only run on the latest 3.7, 3.8, and 3.9 versions of python here? If so it might be worth naming the file as such or adding a comment to that affect.

Reply:
I changed it slightly to more closely mirror the current configuration. My assumption of the intent is just to run fewer versions in PR, so now it will run 2.7, 3.8, 3.9 on linux and 3.6+coverage on windows.